lib/scope.js

```diff
@@ -297,25 +307,34 @@ module.exports = {
 
     let start_url = await scope.page.url();  // so we can later check for navigation
 
-    let winner;
-    winner = await Promise.race([
-      Promise.all([  // Error loads page, so no need to detect to keep things short
-        // Click with no navigation will end immediately
-        // Can't use elem[ scope.activate ](). It works locally, but errors on GitHub
-        // with "Node is not visible or not an HTMLElement". We're not using a custom Chromium version
-        // Unfortunately, this won't catch an invisible element. Hopefully we'd catch it before now...?
-        // "Tap" on mobile is more complex to implement ourselves.
-        // See https://stackoverflow.com/a/56547605/14144258
-        // and other convos around it. We haven't made it public that the device
-        // can customized. Until we do that, we'll just use "click" all the time.
+    let inner;
+    // Error loads page, so no need to detect to keep things short
+    // Click with no navigation will end immediately
+    // Can't use elem[ scope.activate ](). It works locally, but errors on GitHub
+    // with "Node is not visible or not an HTMLElement". We're not using a custom Chromium version
+    // Unfortunately, this won't catch an invisible element. Hopefully we'd catch it before now...?
+    // "Tap" on mobile is more complex to implement ourselves.
+    // See https://stackoverflow.com/a/56547605/14144258
+    // and other convos around it. We haven't made it public that the device
+    // can customized. Until we do that, we'll just use "click" all the time.
```

Comment:
It's a tad complex, so we can probably leave it as is for now. Here are thoughts to maybe make into an issue for future clean up.

Line 311: Think it goes under Promise.race, since it explains (I think?) why we're not racing to detect an invalid input error.

Line 312: Probably goes with the Promise that clicks and navigates.

Lines 313-319: not sure where they should go, so we can leave them here, but they should go somewhere else someday if we can keep an eye out for that. Maybe in our own repo's docs.

Reply:
I moved those first two lines to be in the right places. I think the other lines are fine where they are